linkis-dist/package/db/linkis_ddl.sql

```diff
@@ -257,7 +257,7 @@ CREATE TABLE `linkis_ps_variable_key` (
   `value_type` varchar(50) DEFAULT NULL COMMENT 'Reserved word',
   `value_regex` varchar(100) DEFAULT NULL COMMENT 'Reserved word',
   PRIMARY KEY (`id`),
-  KEY `application_id` (`application_id`)
+  KEY `inx_lpvk_aid` (`application_id`)
```

Comment:
no need to include table name 
 inx_lpvk_aid ->inx_aid

Reply:
Some indexes add table name abbreviations because some indexes have the same name, so table name abbreviations are added to distinguish（部分索引加上表名缩写是因为有的索引名字相同，所以加表名缩写用来区分）